sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -145,7 +144,7 @@
         new MockConsumer<byte[], byte[]>(offsetResetStrategy) {
           // override assign() in order to set offset limits & to save assigned partitions.
           @Override
-          public void assign(final Collection<TopicPartition> assigned) {
```

Comment:
Better for this to stay 'Collection'? I am proposing making kafka-client version to be a maven property that we can set so that it is easy to quickly test with different versions.